ContentForm.jsx

```diff
@@ -44,15 +44,14 @@ export default class ProductForm extends React.Component {
 
   _handleSubmit (event) {
     event.preventDefault();
-    let product = {};
-    product.name = this._nameInput.value;
-    product.release_date = this._releaseDateInput.value;
+    let content = {};
+    content.product_id = this._productIdInput.value;
+    content.source = this._source.value;
```

Comment:
We should stay consistent with our naming since it's a DOM node. `this._sourceInput`